### ISSUE TITLE
Switch to Debian Jessie for our Docker instance

### ISSUE THREAD

**Issue by gsfraley** [Loquius/Loquius#17]

**Comment by DomThePorcupine** [comment]
Does this mean anything to me at the moment?

**Comment by gsfraley** [comment]
Nope!  It's just for Docker, so if you're not using Docker yet, there's
nothing to worry about.

**Comment by gsfraley** [comment]
Althoughhhh, I'm going to assign this to you so you can get setup with and handle the ropes of Docker.  Come get me and we can work on this one together.

**Comment by DomThePorcupine** [comment]
Okay sounds good!
